src/components/calculator-screen.js

```diff
@@ -0,0 +1,8 @@
+function calculatorScreen(screenId,inputId) {
+    return`
+    <div id="${screenId}">
```

Comment:
Why do you need the div and input Id dynamically?
imo it'll be never changed! 

And it's missing to pass the value parameter, isn't it?